gatsby-config.js

```diff
@@ -1,11 +1,24 @@
 const { defaultIcons } = require("gatsby-plugin-manifest/common");
 
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
+
+const {
+  NODE_ENV,
+  URL: NETLIFY_SITE_URL = `https://skynetlabs.com`,
+  DEPLOY_PRIME_URL: NETLIFY_DEPLOY_URL = NETLIFY_SITE_URL,
+  CONTEXT: NETLIFY_ENV = NODE_ENV,
+} = process.env;
+const isNetlifyProduction = NETLIFY_ENV === "production";
+const siteUrl = isNetlifyProduction ? NETLIFY_SITE_URL : NETLIFY_DEPLOY_URL;
+
 module.exports = {
   siteMetadata: {
     title: `Skynet Labs`,
     description: `Skynet Labs is a company behind Skynet - decentralized file sharing and content distribution protocol`,
     author: `Skynet Labs`,
-    siteUrl: `https://skynetlabs.com`,
+    siteUrl,
     image: `https://skynetlabs.com/icons/icon-512x512.png`,
```

Comment:
should this be a relative link since depending on the env we can host on different hostnames ?

Reply:
Right now you can configure it by setting `URL` env variable.

If not set, it will default to skynetlabs.com.

Netlify sets it, so on preview builds the `siteUrl` is `https://deploy-preview-14--musing-kepler-88c11f.netlify.app/` (example).